src/connect4/core.rs

```diff
@@ -83,8 +95,8 @@ impl From<GridPosition> for graphics::Rect {
 impl From<GridPosition> for Point2<f32> {
     fn from(pos: GridPosition) -> Self {
         Point2 {
-            x: (pos.x * BOARD_CELL_SIZE.0 - ((BOARD_CELL_SIZE.0 - (2 * BOARD_DISC_RADIUS)) / 2)) as f32,
-            y: (pos.y * BOARD_CELL_SIZE.1 - ((BOARD_CELL_SIZE.1 - (2 * BOARD_DISC_RADIUS)) / 2)) as f32
+            x: pos.x as f32,
+            y: pos.y as f32
```

Comment:
Tried changing above to make it work but `into()` still gets confused about what's the type, maybe enables using `from::Point2` but it is already similar in-length to `Point2 {x:<>, y:<>}.